module-services/service-cellular/service-cellular/ServiceCellular.hpp

```diff
@@ -81,7 +81,8 @@ class ServiceCellular : public sys::Service
     static const char *serviceName;
 
     /**
-     * @brief Its getting selected SIM card own number.
+     * @brief Its
```

Comment:
I'd rewrite this comment so that it isn't confusing about the "own number". Maybe something like: `@brief Get selected SIM card IMSI number` and then at the end: `@remark see <url> for IMSI explanation`?